examples/svsm-register.rs

```diff
@@ -18,32 +21,29 @@ struct ProxyArgs {
     /// HTTP url to KBS (e.g. http://server:4242)
     #[clap(long)]
     url: String,
-    /// ID of the workload
-    #[clap(long)]
-    workload: String,
-    /// Pre-calculated measurement (hex encoded string - e.g. 8a60c0196d2e9f)
-    #[clap(long)]
-    measurement: String,
-    /// Secret to share with the CVM
-    #[clap(long)]
-    passphrase: String,
 }
 
 fn main() -> anyhow::Result<()> {
     env_logger::init();
+    let test_measurement = [0u8; 1]; // A testing, non-usable measurement.
```

Comment:
The intent of this tool is to be used to register also a real measurement. I'd leave the arguments and maybe set a default value for testing.